content/influxdb/cloud/notebooks/clean_data.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Normalize data with notebooks
+description:
+weight: 102
+influxdb/v2.0/tags:
+menu:
+  influxdb_2_0:
+    name: Normalize data with notebooks
+    parent: Notebooks
+---
+{{% note %}}
+**Notebooks is currently an early access feature.**
+[Submit a request](https://w2.influxdata.com/notebooks-early-access/ ) to be added to the queue, and we will send you a confirmation when you’ve been added to early access.
+{{% /note %}}
+
+Make it easier to compare data from different places by standardizing or normalizing it. This example walks through creating a notebook with NOAA water database sample data that makes it easier to view water level and water temperature measurements alongside one another using the [`map()`](/influxdb/cloud/reference/flux/stdlib/built-in/transformations/map/) function and writing it to a new bucket.
```

Comment:
Make it easier to compare numeric metrics at different scales by normalizing values to be between 0 and 1.